github/util.py

```diff
@@ -116,10 +116,11 @@ def retrieve_email_address(self, user_name):
 
 
 def _create_github_api_object(
-    github_cfg: 'GithubConfig'
+    github_cfg: 'GithubConfig',
+    auth_token: str=None
```

Comment:
it should be documented that the auth_token argument will overwrite the one from github_cfg.

Although it might be the cleaner approach to patch-in the auth_token into the github_cfg's credentials object and not change this function

Reply:
Documentation is done. 
Just patching/exchange the token from the created `github_cfg` object is not possible. One would need to create a new login session with the new token

Reply:
that's true. but this does not actually contradict my proposal. in this function's first line, a github_api object is created from the passed github_cfg. if the token would have been patched before calling this function, this would subsequently lead to a github_api object using the patched token